README.md

```diff
@@ -65,11 +46,7 @@ Give a ⭐️ if you like this project!
 ## Acknowledgments
 
 - Hat tip to anyone whose code was used
-- Inspiration
-- etc
 
 ## 📝 License
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] OPTIONAL Please, consider fixing the broken link to the MIT.md file
![image](https://user-images.githubusercontent.com/49064106/193233350-3154bef9-73a9-4297-8af5-c13ec158a9c1.png)